dj_rql/filter_cls.py

```diff
@@ -40,6 +39,12 @@
 
 from lark.exceptions import LarkError
 
+try:
```

Comment:
Probably for the future it will be better to define some variable, like if DRF is available, and rely on it... Because now it is really weird. Especially when we try to import from our library, not from drf itself